### ISSUE TITLE
dependency not found  ~\content in ./.nuxt/plugins.markdownComponents.96959a12.js 

### ISSUE THREAD

**Issue by alterhu2020** [nuxt-community/nuxtent-module#78]
Hi Team,
 I use this module for my person blog, and  I check to use the example located here: 
[multiple-content-types](https://github.com/nuxt-community/nuxtent-module/tree/master/examples/multiple-content-types)

After installing dependency using command `npm install` ,and Run the command `npm run dev` ,It shows below exception message:
```

This dependency was not found:

* ~\content in ./.nuxt/plugins.markdownComponents.96959a12.js

To install it, you can run: npm install --save ~\content

```
![image](https://user-images.githubusercontent.com/5029046/30783123-7a2a6c44-a170-11e7-8ace-63d546ad433c.png)

The nuxt version is: 1.0.0-rc11
The nuxtent version is: 0.2.79
All they are latest version.

Guy, do you have any update for this critical issue , which had totally break the `nuxtent` .
thanks for your reply. also thanks for your great effort working on this module. great plugin...

**Comment by alterhu2020** [comment]
@medfreeman , It seems that totally `nuxtent` not worked now , It's too bad now.

**Comment by alidcastano** [comment]
I just updated the [ency documentation](https://github.com/encyjs/docs) to using latest version of nuxt and nuxtent.

Can you try removing your node_modules and installing again. Thanks

**Comment by medfreeman** [comment]
@alterhu2020 please provide your os flavor / version and npm version, it helps reproducing the issues.

I guess you're on windows. i got the same error.
Fixing this in a moment.
Will also add windows and linux ci builds probably today, to be able to release safely in the future.

**Comment by alidcastano** [comment]
Yeah, I'm on mac and did not get that error, so that makes sense

**Comment by medfreeman** [comment]
@alidcastano you can release the fix.
Or if you want to give me publish rights on npm, that also works, whatever you prefer.

**Comment by alidcastano** [comment]
@alterhu2020 Thanks for opening this issue. We will be releasing the fix soon in the next release.

Also, once you have your site running in nuxtent, feel free to add it to readme!

**Comment by alterhu2020** [comment]
This change don't make any sense ,it still met the same exception after upgrade `nuxtent` to `nuxtent@0.2.80`  .

1. The OS version and platform is:  Windows 7 Enterprise SP1 64 bit system.
2. The npm version is:  V5.4.2
3. The nodejs version is:  V8.5.0

You can easily to reproduce this issue , just download the example project ,location: https://github.com/nuxt-community/nuxtent-module/tree/master/examples/multiple-content-types

If you need more infos, please let me know ,thanks for your team.

![image](https://user-images.githubusercontent.com/5029046/30791869-a4c45dda-a16a-11e7-877d-74e054e22b01.png)

**Comment by alterhu2020** [comment]
@alidcastano, @medfreeman , Please help to reopen this issue again.

**Comment by alterhu2020** [comment]
also when i downgrade to `nuxtent@0.2.77` I  found that the `axios` request met the CROS issue, like below from console:
```
XMLHttpRequest cannot load http://0.0.0.0:3000/blog/?. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:3000' is therefore not allowed access. The response had HTTP status code 502
```
I think `nuxt` had added the  'Access-Control-Allow-Origin':* header , why it happened for `axios` in `nuxtent`.
 
I can use `@nuxtjs/axios` without this CROS issue totally  when use it separately.

**Comment by medfreeman** [comment]
The issue was closed in v1.0.0
